src/initial_conditions/atmos_state.jl

```diff
@@ -102,12 +102,9 @@ moisture_variables(ls, ::NonEquilMoistModel) = (;
     ρq_ice = ls.ρ * TD.ice_specific_humidity(ls.thermo_params, ls.thermo_state),
 )
 
-# TODO: Remove perf_mode. Currently, adding tracers hurts performance.
-precip_variables(ls, ::NoPrecipitation, ::PerfStandard) = (;)
-precip_variables(ls, ::Microphysics0Moment, ::PerfStandard) = (;)
-precip_variables(ls, ::Microphysics1Moment, ::PerfStandard) =
-    (; ρq_rai = zero(eltype(ls)), ρq_sno = zero(eltype(ls)))
```

Comment:
@Sbozzolo, this deleted function is where the tracers are added. It's difficult to see in the diff, but in the main branch it looks like

```julia
precip_variables(ls, _, ::PerfExperimental) =
    (; ρq_rai = zero(eltype(ls)), ρq_sno = zero(eltype(ls)))
```

Reply:
Aren't the two identical?

Reply:
No, the second argument is more generic, so it catches basically any precip model (even if it's nothing).